index.html

```diff
@@ -165,5 +165,21 @@ <h4 class="end-text">LETS'S CONNECT</h4>
             <hr>
         </article>
     </main>
+    <footer class='container'>
+        <article >
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/xayvwlyd" method="post" class="contacts-form">
+                <label for="username"></label>
+                <input id="username" type="text" placeholder="Username" required name="username">
```

Comment:
- [ ] For this input, you are supposed to include the `maxlength=30` attribute. This will enhance validations by ensuring that the user does not input more that 30 characters in this input section.

Reply:
I put it but it gives me a linter error so , and the error message ask me to put it in CSS style it's confusing